internal/pkg/skuba/addons/dex.go

```diff
@@ -163,8 +163,10 @@ metadata:
   namespace: kube-system
   labels:
     app: oidc-dex
+    caasp.suse.com/skuba-replica-ha: "true"
 spec:
   replicas: 3
+  revisionHistoryLimit: 0
```

Comment:
Any purpose for a specific value? not default one.

Reply:
Default is set to 10. For every replica update patch will remain in the revision history and shows up in `kubectl get`. This configuration will ensure no revision history for dex and gangway is shown. This could be removed if we want to keep the unncessary revisions in history.

Reply:
I am fine for now. At least for now no addon rollback scenario.

Reply:
Related to https://bugzilla.suse.com/show_bug.cgi?id=1162651

Reply:
We can take this time to resolve this issue. No strong opinion about what the value is, so just take the suggestion from the bug to lower down the history number first.

@c3y1huang could you help change to `3` then we can align with other addons later for solving the https://bugzilla.suse.com/show_bug.cgi?id=1162651.

Reply:
done